plugins/plugins.vim

```diff
@@ -8,14 +8,20 @@ Plug 'morhetz/gruvbox'
 
 Plug 'vim-airline/vim-airline'
 
-Plug 'scrooloose/nerdtree'
-
 Plug 'vim-syntastic/syntastic'
 
 Plug 'mkitt/tabline.vim'
 
 
 " ----------------------------------------------------------------
+"  Buffer plugins
```

Comment:
Why Buffer plugins ? How is this linked to buffers ? Buffer definition : A buffer is a file loaded into memory for editing. All opened files are associated with a buffer. -> Navigation ou files plugins ?